server/bin/bind-performance.rb

```diff
@@ -26,9 +29,9 @@ def time_rand from = 0.0, to = Time.now
   Time.at(from + rand * (to.to_f - from.to_f))
 end
 
-cp = Candlepin.new(ADMIN_USERNAME, ADMIN_PASSWORD, nil, nil, HOST, PORT)
+@cp = Candlepin.new(ADMIN_USERNAME, ADMIN_PASSWORD, nil, nil, HOST, PORT)
```

Comment:
@Ceiu also, I tested all the scripts, and this one worked. only one of them did not work I think, and that was broken for un-related reasons, I figured its stale code at this point not worth fixing.